WordPress/src/main/res/values/strings.xml

```diff
@@ -2126,8 +2126,8 @@
     <!-- accessibility content desciptions -->
     <string name="reader_btn_remove_filter_content_description">Remove the current filter</string>
     <string name="reader_btn_select_filter_content_description">Select a Site or Tag to filter posts</string>
```

Comment:
💡 It looks like this uses "site" and "tag" but at the same time it's an unused resource so maybe we can just get rid of it in this `main` `strings.xml` file?